### PR TITLE
Prepare to release 3.6.2

### PR DESCRIPTION
**Pull request by heyams**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## Version 3.6.2 GA (10/23/2024)
+
+### Enhancements
+
+* Update azure-monitor-opentelemetry-exporter to 1.0.0-beta.31
+  ([#3932](https://github.com/microsoft/ApplicationInsights-Java/pull/3932))
```
Comment (trask) [inline CHANGELOG.md:8]:
```suggestion
* Update to OpenTelemetry Java Instrumentation 2.9.0
  ([#3918](https://github.com/microsoft/ApplicationInsights-Java/pull/3918))

### Bug fixes

* Restore missing self-diagnostic log
  ([Azure/azure-sdk-for-java#42451](https://github.com/Azure/azure-sdk-for-java/pull/42451))
```

Comment (heyams) [reply CHANGELOG.md:8]: why does it need to repeat the same bug fixes in multiple places?

Comment (heyams) [reply CHANGELOG.md:8]: exporter update includes those two. i don't think we need to repeat it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+## Version 3.6.2 GA (10/23/2024)
+
+### Enhancements
+
+* Update to OpenTelemetry Java Instrumentation [2.9.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v2.9.0)
+  ([#3917](https://github.com/microsoft/ApplicationInsights-Java/pull/3917))
+
+* Update azure-monitor-opentelemetry-exporter to [1.0.0-beta.31](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md#100-beta31-2024-10-23)
+  ([#3917](https://github.com/microsoft/ApplicationInsights-Java/pull/3932))
+  Note: This version of the exporter includes a fix for [restoring slef-diagnostic logging](https://github.com/Azure/azure-sdk-for-java/pull/42451)
```
Comment (trask) [inline CHANGELOG.md:12]:
```suggestion
  Note: This version of the exporter includes a fix for [restoring self-diagnostic logging](https://github.com/Azure/azure-sdk-for-java/pull/42451)
```

**Review by trask — APPROVED** [review]

**Pull request merged**